olive/model/model.py

```diff
@@ -9,29 +9,29 @@
 from abc import ABC, abstractmethod
```

Comment:
nit: this file could be be merged with __init__.py

Reply:
Sounds good. Will do.